Humbertino_Avilez/CRUD-EXCEPTION/Vet_Patitas/src/main/java/com/bolsadeideas/springboot/bootcamp/app/dao/AnimalDao.java

```diff
@@ -0,0 +1,9 @@
+package com.bolsadeideas.springboot.bootcamp.app.dao;
+
+import org.springframework.data.repository.CrudRepository;
+
+import com.bolsadeideas.springboot.bootcamp.app.entity.Animal;
+
+public interface AnimalDao extends CrudRepository<Animal, Long>{
```

Comment:
Why use CrudRepository and not JPARepository? tell my the difference.

Reply:
According to the literature consulted: CrudRepository mainly provides CRUD functions and JPARepository inherits from PagingAndSortingRepository in turn, PagingAndSortingRepository inherits CrudRepository, So if we dont need extra functions (JPA persistence technology) we must use CrudRepository  or PagingAndSortingRepository (as its name suggests sorting y paging)